doc/nrf/releases_and_maturity/releases/release-notes-changelog.rst

```diff
@@ -128,6 +128,9 @@ Security
 
   * The :ref:`ug_tfm_logging` page with more details about how to configure logging on the same UART instance as the application for nRF5340 and nRF91 Series devices.
   * The :ref:`crypto_drivers` page with more details about the driver selection process.
+  * The :ref:`ug_crypto_supported_features` page with updated values for the supported cryptographic operations and algorithms.
+    The page now only lists features and algorithms that are supported by at least one driver.
+    If no drivers support a given feature, the page does not list it.
```

Comment:
Line 133: Do we need this sentence as I think its clear by mentioning "at least one driver." in the previous sentence?